### ISSUE TITLE
Non-nullable fields should have 'presence' validation

### ISSUE THREAD

**Issue by timwis** [JarvusInnovations/jarvus-postgrest-apikit#4]
As you've suggested [there appears to be a bug](https://www.sencha.com/forum/showthread.php?290842) (or missing feature) preventing you from adding validation after a model's been initialized.

**Comment by themightychris** [comment]
One quick note -- the presence validation should be added when a field is
both non-nullable AND has no db-driven default

On Fri, Aug 14, 2015 at 7:12 AM, Tim Wisniewski notifications@github.com
wrote:

> As you've suggested there appears to be a bug
> https://www.sencha.com/forum/showthread.php?290842 (or missing feature)
> preventing you from adding validation after a model's been initialized.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/JarvusInnovations/jarvus-postgrest-apikit/issues/4.

## 

Chris Alfano
Chief Developer
Jarv.us Innovations http://jarv.us

(e) chris@jarv.us (m) 267-918-6166

**Comment by timwis** [comment]
Good point. When you said you'd "work on this," was that before you knew it was a known Ext.js bug? Or do you still think it's possible to fix it?
